lucene/analysis/kuromoji/src/test/org/apache/lucene/analysis/ja/dict/TokenInfoDictionaryTest.java

```diff
@@ -35,6 +35,7 @@
 
 import static org.apache.lucene.analysis.ja.dict.BinaryDictionary.ResourceScheme;
 
+//nocommit
```

Comment:
"test-tools" shouldn't be a task. I don't know what it was doing but it should be a regular test, perhaps grouped under a test group (and enabled by a property).